src/Quadrature/gaussquad_tri_table.jl

```diff
@@ -5,25 +5,25 @@
 #
 # Quadrature rules for orders 9 to 20 have been obtained using the
 # basix.make_quadrature(basix.CellType.triangle, n) calls of the
-# FEniCS / basix python package
+# FEniCS / basix python package, which corresponds to Gauss-Jacobi rules.
 #
 # see
 # https://docs.fenicsproject.org/basix/main/python/_autosummary/basix.html?highlight=quadraturetype#basix.make_quadrature
```

Comment:
I really struggle with finding the reference here. From what I understand it should be https://www.degruyter.com/document/doi/10.1515/crll.1826.1.301/html but the formula is slightly different ($w(x)=(x-1)^\alpha$?) from the formula I find in modern literature on ($w(x)=(x-1)^\alpha (x+1)^\beta$, e.g. https://epubs.siam.org/doi/pdf/10.1137/120889873).

Reply:
But these are references for line intervals? Or do I miss something? 

Seems like dunavant provides rules up to `n=20`. (https://onlinelibrary.wiley.com/doi/epdf/10.1002/nme.1620210612) (Although some are based on previous works, as noted on the bottom of page 1131). 

Since higher than 7 were added in #514 and probably no one are saving quadrature data for such high orders, I think it could make sense to only use `dunavant`. But due to the different coordinate systems used in the papers and in the code, I find it hard to check where the points come from. 
How did you check this to figure out that starting at n = 9 this is gauss_jacobi?